### PR TITLE
Feature: dependency version bump

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

**Comment by lsh-0** [comment]
build has failed because of some change in feedgen between 0.5 and 0.7, investigating

**Comment by lsh-0** [comment]
fixed. in 0.6 feedgen decided `.add_entry` should prepend rather than append to the list of entries. made the two tests clearer as well

File: requirements.txt [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-python-slugify==1.2.4
-pytz==2017.2
-requests==2.13.0
+django-markdown2==0.3.1
+et3==1.5.1
+feedgen==0.7.0
+green==2.13.0
+ipython-genutils==0.2.0
+kids.cache==0.0.7
+newrelic==4.4.1.104
```
Comment (giorgiosironi) [inline requirements.txt]: will keep :eyes: on the upgrade but it has gone well elsewhere

File: src/observer/rss.py [hunk 2]
```diff
@@ -48,7 +48,9 @@ def mkfeed(report):
     return fg
 
 def add_entry(fg, item):
-    entry = fg.add_entry()
+    # default order of insertion changed in 0.6 to 'prepend'
+    # https://github.com/lkiesow/python-feedgen/blob/1b301f67adf4e2f0367579a6c41f72ee524b9380/feedgen/feed.py#L999
+    entry = fg.add_entry(order='append')
```
Comment (giorgiosironi) [inline src/observer/rss.py]: very nice catch

**Pull request merged**